src/main/java/net/minecraftforge/event/level/ChunkEvent.java

```diff
@@ -58,9 +58,30 @@ public ChunkAccess getChunk()
      **/
     public static class Load extends ChunkEvent
     {
+        private final boolean newChunk;
+
+        @Deprecated
         public Load(ChunkAccess chunk)
+        {
+            this(chunk, false);
+        }
+
+        public Load(ChunkAccess chunk, boolean newChunk)
         {
             super(chunk);
+            this.newChunk = newChunk;
+        }
+
+        /**
+         * Check whether the Chunk is newly generated, and being loaded for the first time.
+         * <br>
+         * Will only ever return {@code true} on the {@link net.minecraftforge.fml.LogicalSide#SERVER logical server}.
```

Comment:
Prefer new paragraphs instead of line breaks. Use `{@linkplain}` instead of `{@link}` when the label of the tag should be rendered as plain text.
```suggestion
         *
         * <p>This will only ever return {@code true} on the {@linkplain net.minecraftforge.fml.LogicalSide#SERVER logical server}.</p>
```